ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -60,6 +63,13 @@ const CREDIT_LIMIT_DEFAULT_MIN = 300;
 const CREDIT_LIMIT_DEFAULT_MAX = 800;
 const CREDIT_LIMIT_DEFAULT_STEP = 50;
 
+const getUserStatus = (profile: Profile) => {
+  const {accessTierShortNames} = profile;
+  return (hasRegisteredAccess(accessTierShortNames))
+      ? () => <div style={{color: colors.success}}>Enabled</div>
+      : () => <div style={{color: colors.danger}}>Disabled</div>
```

Comment:
nit: semicolon? Does linter not complain here?

Reply:
I think our linter is on vacation, actually.  Something about the Angular-React and tslint/eslint changes.  I suspect we'll need to re-lint everything added over these few weeks.

thanks for the catch.